### ISSUE TITLE
Package xcolor still required in version 1.1.0

### ISSUE THREAD

**Issue by xelra** [haozhu233/kableExtra#436]
I'm just getting started with knitr and kableExtra. I was reading the documentation very carefully and I was under the impression that with `kableExtra >= 1.0`, the xcolor package was not to be used anymore.

I'm using CoCalc (formerly SageMathCloud) and checked with `sessionInfo()` that it's using `knitr_1.25` and `kableExtra_1.1.0`. 

I set `options(kableExtra.latex.load_packages = FALSE)` and loaded all the packages from page 4 of the documentation manually. All but xcolor. This worked fine. As soon as I load xcolor, I'm getting `Package xcolor Warning: Incompatible color definition on input line xxx`.

But without xcolor, the `striped` option doesn't work and it looks like this instead:
![image](https://user-images.githubusercontent.com/908568/67699835-1811f800-f9ad-11e9-9d71-8d0f36741ac1.png)

I was under the impression that `striped` should only rely on the package colortbl anymore. Did I do something wrong or is this a regression?

**Comment by haozhu233** [comment]
I re-read what I wrote in the doc and realized that I made a mistake there. Thanks for bringing that into my attention. I will update the doc.

Right now `kableExtra` is still calling `xcolor` to define stripe color (only purpose).
What has been changed is that in the past, we were using the `[table]` option of xcolor, which may cause a conflict with the newer version of `fancyvbr` (the author updated that package last year after 10 years). Therefore, as you noted, `xcolor` is still loaded in the package list on page 4. What was removed was the [table] option. 

Since we removed the [table] option, in order to deal with striped rows, we started to use `colortbl` to do it in a different way. In fact,  I was debating whether to completely remove the xcolor dependency or just solve the conflict. Since I didn't want to cause more conflicts for people's existing code, I picked the later one. This is the whole story. 

I'm not sure why you are getting that error after load xcolor. I suggest to look into your other color definitions and make sure it's compatible with xcolor.

**Comment by xelra** [comment]
@haozhu233 
I see. That explains it. Looking forward to having the xcolor dependency completely removed.

I've looked further into where the `Package xcolor Warning: Incompatible color definition on input line xxx` warning is coming from and I noticed that I even have this warning in an absolute minimal file with the table from the project's README.

The culprit seems to be this line in the expansion
```
\definecolor{shadecolor}{rgb}{0.969, 0.969, 0.969}\color{fgcolor}\begin{kframe}
```
Specifically `\color{fgcolor}`. I think it doesn't know what `fgcolor` is. It's probably undefined?
